sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md

```diff
@@ -65,7 +65,7 @@ Example 3 creates a new site collection for the current company with specified s
 
 Specifies the version of templates to use when you are creating a new site collection. This value sets the initial CompatibilityLevel value for the site collection. When this parameter is not specified, the CompatibilityLevel will be by default the highest possible version for the Web application.
```

Comment:
I would replace this explainer text with something akin to:

"This parameter no longer has any effect and only accepts a value of '15'".